packages/e2e/test/environment.ts

```diff
@@ -1,29 +1,84 @@
 import * as envalid from 'envalid';
 
-export const getEnv = (override: NodeJS.ProcessEnv = {}) =>
+/**
+ * Shared collection of all the environment configuration variables with their constraints
```

Comment:
```suggestion
 * Shared across all tests
```
We get the intent from the rename to `validators`.